application/src/main/java/org/togetherjava/tjbot/features/basic/MemberCounter.java

```diff
@@ -0,0 +1,48 @@
+package org.togetherjava.tjbot.features.basic;
+
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.channel.concrete.Category;
+
+import org.togetherjava.tjbot.config.Config;
+import org.togetherjava.tjbot.features.Routine;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Integer.parseInt;
+
+public class MemberCounter implements Routine {
+    private final String wordsInCategory;
+
+    private final String updateMemberCountEveryHour;
+
+    public MemberCounter(Config config) {
+        this.wordsInCategory = config.getMemberCountCategoryName();
+        this.updateMemberCountEveryHour = config.getupdateMemberCountEveryHour();
+    }
+
+    private void updateCategoryName(Category category) {
+        int totalMemberCount = category.getGuild().getMemberCount();
+        String basename = category.getName();
+        category.getManager()
+            .setName("%s - %d Members".formatted(basename, totalMemberCount))
+            .queue();
+    }
+
+    @Override
+    public Schedule createSchedule() {
+        return new Schedule(ScheduleMode.FIXED_RATE, 0, parseInt(updateMemberCountEveryHour),
```

Comment:
just hardcore the number here and get rid of the var and config. u can go for one day

Reply:
not done

Reply:
Done